core/provider/provider_ep2.cpp

```diff
@@ -132,7 +132,7 @@ void BaseProvider::ConsolePrint(const char *str)
 void BaseProvider::Notify_DLLInit_Pre(CreateInterfaceFn engineFactory, 
 									  CreateInterfaceFn serverFactory)
 {
-#if SOURCE_ENGINE == SE_TF2 || SOURCE_ENGINE == SE_CSS || SOURCE_ENGINE == SE_DODS || SOURCE_ENGINE == SE_HL2DM || SOURCE_ENGINE == SE_SDK2013
+#if SOURCE_ENGINE == SE_TF2 || SOURCE_ENGINE == SE_CSS || SOURCE_ENGINE == SE_DODS || SOURCE_ENGINE == SE_HL2DM || SOURCE_ENGINE == SE_SDK2013 || SOURCE_ENGINE == SE_PVKII
```

Comment:
This scope can actually be changed to only check against SE_SDK2013. We initially put it in at a point where TF2 had the change, but CS:S/DoD:S/HL2DM hadn't had an update yet to sync them up (but we assumed they eventually would, and they did).

Reply:
Is that the same case here in SourceMod? https://github.com/Octoshark/sourcemod-pvkii/blob/master/core/sourcemm_api.cpp#L73

Reply:
And here: https://github.com/Octoshark/sourcemod-pvkii/blob/master/public/smsdk_ext.cpp#L344

Reply:
> Is that the same case here in SourceMod?

> And here

Yes to both